cloudnet-lib/src/main/java/de/dytanic/cloudnet/lib/player/permission/DefaultPermissionGroup.java

```diff
@@ -14,6 +14,6 @@
 
     public DefaultPermissionGroup(String name)
     {
-        super(name, "§e", "§f", "§7", 98, 0, false, new HashMap<>(), new HashMap<>(), new HashMap<>(), new ArrayList<>());
+        super(name, "§c","§e", "§f", "§7", 98, 0, false, new HashMap<>(), new HashMap<>(), new HashMap<>(), new ArrayList<>());
```

Comment:
```suggestion
        super(name, "§c", "§e", "§f", "§7", 98, 0, false, new HashMap<>(), new HashMap<>(), new HashMap<>(), new ArrayList<>());
```

Please format your code properly